src/Language/ASKEE/Model/Basics.hs

```diff
@@ -108,6 +108,7 @@ data ModelType =
   | GrometPrtType
   | GrometPncType
   | GrometFnetType
+  | LatexType
```

Comment:
Despite the fact that it used to be a model, I don't think Latex should be thought of a type of model for now.  It should mainly just be a pretty printing target for `DiffEq` - I think we can accomplish everything necessary for this change without adding more flavors of Model.  

I'd suggest removing this and things depending on it.

Reply:
Good to know. I've removed all of the gunk associated with `LatexType`.